.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
We could remove this file if it's empty

Reply:
A lot of people set there IDE to format on save if a prettier file is in the project.